src/contrib/serializers/Akka.Serialization.TestKit/AkkaSerializationSpec.cs

```diff
@@ -336,11 +336,11 @@ public void CanSerializeActorPathContainer()
         [Fact]
         public void CanSerializeSingletonMessages()
         {
-            var message = Terminate.Instance;
+            var message = PoisonPill.Instance;
```

Comment:
`Terminate` is no longer a singleton message type, therefore not valid for this test.